test/compiler.ts

```diff
@@ -8,6 +8,12 @@ import wrapper from '../wrapper';
 
 const noRemoteVersions = (process.argv.indexOf('--no-remote-versions') >= 0);
 
+// use a function that creates a temporary dir instead of /tmp
+function createTempDir () {
+  const tmpDir = require('tmp').dirSync({ unsafeCleanup: true });
```

Comment:
Please use an import statement above and not `require` here.

Reply:
Ok